Cargo.toml

```diff
@@ -18,6 +18,12 @@ license = { workspace = true }
 authors = { workspace = true }
 homepage = { workspace = true }
 
+[features]
+# Static linking is default.
+default = ["static"]
+dynamic = ["sdl2/use-pkgconfig"]
+static = ["sdl2/bundled", "sdl2/static-link"]
```

Comment:
Hmm, here you have distinct “dynamic” and “static” features, but in one of the build.rs you're doing an if-not-dynamic check as if that implies static, but that's not guaranteed right? I think it'd be cleaner to just have a “static” feature, and if it's not present, then assume dynamic.

Also, maybe there could be separate “SDL2 static” and “OpenAL static” features? (By the way, wouldn't you have to update the Cargo.toml files for the inner crates for the static linking feature to pass through? Did you test the current setup actually works?)

Reply:
Yeah, that's a good idea. I removed the dynamic feature in my commit I'm about to push. I made a static feature in the `openal_soft` crate (and the `dynarmic_wrapper` crate, not sure if it's used there), and the static feature enables them by default. 

If you only need to dynamically link SDL2 and statically link OpenAL, you can just do something like `cargo build --no-default-features --features touchHLE_openal_soft_wrapper/static`.